.github/workflows/continuous-integration.yml

```
@@ -79,7 +79,7 @@ jobs:
           fetch-depth: 2
 
       - name: "Download coverage files"
-        uses: "actions/download-artifact@v3"
+        uses: "actions/download-artifact@v4.1.7"
```

We don't want to pin the action, do we?

```suggestion
        uses: "actions/download-artifact@v4"
```

---

You're right, let's avoid generating too many PRs